doc/MANUAL.txt

```diff
@@ -32,6 +32,40 @@ Most commands in testr have comprehensive online help, and the commands::
 
 Will be useful to explore the system.
 
+Concepts
+~~~~~~~~
+
+Repository
+----------
+
+A store of the output from tests that were run, which can be datamined.
+Defaults to `./.testrepository`.
+
+Subunit stream
+--------------
+
+A structured format for representing the activity that occurs during a test
+run. See the subunit project documentation for more information.
+
+Test profile
+------------
+
+A particular configuration that tests are being run under.
+
+Test id
+-------
+
+The identifier for a test. These should uniquely identify a single test that
+is going to be run. Id's are simple unicode strings that may contain
```

Comment:
Nit: the apostrophe in "Id's" is non-standard: it's neither an abbreviation nor a possession. I'd suggest settling on "ID" singular and "IDs" plural.